### ISSUE TITLE
SoundCloud Player not accessible to the hard of hearing

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-gallery#71]
https://dontbelievethehype.co.nz/tourdiaries/asia/east-asia/russia/13/exploring-irkutsk/

The supermarket audio is accompanied by a photo which reads "CHANGE BATTERY". This is a generic placeholder and does not lend any context to the audio track. There is a short caption below the player, but it is not very descriptive.

The SoundCloud player does not support transcripts, synced or otherwise.

Add a better image, or change the audio to a Vimeo video, and add a synced transcript which describes the various sounds.
